Enable CPU backend on CI pipeline

The CI run failed with the CPU device on the current codebase. This issue just records the steps to reproduce these errors and we should fix them with some PRs.

1. Launch development Docker container

    ``` bash
    docker pull bladedisc/bladedisc:latest-devel-cuda10.0
    docker run --rm -it -v $PWD:/workspace -w /workspace bladedisc/bladedisc:latest-devel-cuda10.0 bash
    ```

1. Execute the command to run unit tests 

    ``` bash
    python scripts/python/tao_build.py /opt/venv_disc/ -s configure --bridge-gcc default --compiler-gcc default --cpu_only
    python scripts/python/tao_build.py /opt/venv_disc/ -s build_tao_compiler --cpu_only
    python scripts/python/tao_build.py /opt/venv_disc/ -s test_tao_compiler --cpu_only 
    ```

There are 7 unit tests that run failed totally, the snippet error message of `sub.cc.test` is as follows, I also upload the error logs file [here](https://github.com/alibaba/BladeDISC/files/7806676/test.log)


``` text
object file to shared library command: gcc --shared -o /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/_tmp/f0a918c4823924815dc3c918baef281fImplicitBroadcast2DF32.so /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/_tmp/f0a918c4823924815dc3c918baef281fImplicitBroadcast2DF32.so.o
/usr/bin/ld: /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/_tmp/f0a918c4823924815dc3c918baef281fImplicitBroadcast2DF32.so: version node not found for symbol packed_@K&y^KV
/usr/bin/ld: failed to set dynamic section sizes: Bad value
collect2: error: ld returned 1 exit status
save shared lib file to : /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/_tmp/f0a918c4823924815dc3c918baef281fImplicitBroadcast2DF32.so
```

The whole building error message is as the following:

``` text
INFO: Elapsed time: 53.235s, Critical Path: 44.90s
INFO: 116 processes: 30 remote cache hit, 139 local.
INFO: Build completed, 7 tests FAILED, 116 total actions
//tensorflow/compiler/mlir/disc/tests/regression:empty_tensor.cc.test    PASSED in 3.5s
//tensorflow/compiler/mlir/disc/tests/regression:io_forwarding.cc.test   PASSED in 2.3s
//tensorflow/compiler/mlir/disc/tests/regression:multi_cc.cc.test        PASSED in 1.9s
//tensorflow/compiler/mlir/disc/tests/regression:uint_cast.cc.test       PASSED in 1.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:abs.cc.test         PASSED in 4.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:add.cc.test         PASSED in 4.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:add_n.cc.test       PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:add_v2.cc.test      PASSED in 4.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:all.cc.test         PASSED in 3.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:any.cc.test         PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:batch_matmul.cc.test PASSED in 1.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:batch_matmul_v2.cc.test PASSED in 2.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:bias_add.cc.test    PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:broadcast_to.cc.test PASSED in 3.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:cast.cc.test        PASSED in 2.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:ceil.cc.test        PASSED in 3.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:concat_v2.cc.test   PASSED in 3.5s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:const.cc.test       PASSED in 3.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:conv2d.cc.test      PASSED in 2.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:conv2d_backprop_filter.cc.test PASSED in 1.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:conv2d_backprop_input.cc.test PASSED in 1.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:dynamic_stitch.cc.test PASSED in 4.5s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:equal.cc.test       PASSED in 5.3s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:erf.cc.test         PASSED in 12.3s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:exp.cc.test         PASSED in 3.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:expand_dims.cc.test PASSED in 4.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:fill.cc.test        PASSED in 3.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:floor.cc.test       PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:floor_mod.cc.test   PASSED in 8.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:gather_nd.cc.test   PASSED in 3.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:gather_v2.cc.test   PASSED in 5.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:greater_equal.cc.test PASSED in 4.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:identity.cc.test    PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:isfinite.cc.test    PASSED in 3.7s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:less_equal.cc.test  PASSED in 5.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:log.cc.test         PASSED in 4.1s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:logical_and.cc.test PASSED in 4.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:logical_not.cc.test PASSED in 3.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:logical_or.cc.test  PASSED in 4.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:matmul.cc.test      PASSED in 2.1s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:minimum.cc.test     PASSED in 5.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:mul.cc.test         PASSED in 4.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:neg.cc.test         PASSED in 4.1s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:pack.cc.test        PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:pow.cc.test         PASSED in 5.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:prod.cc.test        PASSED in 8.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:random_uniform.cc.test PASSED in 1.7s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:range.cc.test       PASSED in 3.5s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:relu.cc.test        PASSED in 3.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:relu_grad.cc.test   PASSED in 3.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:reshape.cc.test     PASSED in 4.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:reverse.cc.test     PASSED in 3.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:round.cc.test       PASSED in 6.7s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:rsqrt.cc.test       PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:select.cc.test      PASSED in 4.2s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:shape.cc.test       PASSED in 3.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:sigmoid.cc.test     PASSED in 4.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:sign.cc.test        PASSED in 4.3s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:slice.cc.test       PASSED in 4.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:snapshot.cc.test    PASSED in 4.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:split.cc.test       PASSED in 3.8s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:squared_difference.cc.test PASSED in 6.0s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:squeeze.cc.test     PASSED in 3.9s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:strided_slice.cc.test PASSED in 7.5s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:tanh.cc.test        PASSED in 4.3s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:tile.cc.test        PASSED in 4.6s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:topk.cc.test        PASSED in 1.7s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:transpose.cc.test   PASSED in 3.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:unpack.cc.test      PASSED in 4.3s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:zeros_like.cc.test  PASSED in 3.6s
//tensorflow/compiler/mlir/disc/transforms/tests:conv-rewrite.mlir.test  PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:demo.mlir.test          PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-assign-kernel-name.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-assign-memory-space.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-const-to-ral.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-convert-tensor-to-std.mlir.test PASSED in 0.3s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-flatten-memref-access.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-hlo-legalize-to-lhlo.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-input-inline-fusion.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-legalize-printf-to-llvm.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-lower-to-library-call.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-math-approximation.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-memref-canonicalizer.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-ral-legalize-alloc-dealloc-to-llvm.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-remove-dead-buffer.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-shape-simplifier-tie-shape.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:disc-shape-simplifier.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:dot_rewriter.mlir.test  PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:dynamic-reshape-canonicalizer.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:element_type_converter.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:gpu_conv_padding.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:lhlo-input-inline-fusion.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:map-parallel-loops.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:memref-load-cse.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:mhlo_mark_shape_calc_op.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:mhlo_place_ops.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:parallel-loop-tiling-inbound-check.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:revise-kernel-outlining.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:revise_args.mlir.test   PASSED in 0.4s
//tensorflow/compiler/mlir/disc/transforms/tests:specialize_fusion_with_speculation.mlir.test PASSED in 0.3s
//tensorflow/compiler/mlir/disc/transforms/tests:split_large_ops.mlir.test PASSED in 0.4s
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:floor_div.cc.test    FLAKY, failed in 1 out of 2 in 5.5s
  Stats over 2 runs: max = 5.5s, min = 5.1s, avg = 5.3s, dev = 0.2s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/floor_div.cc.test/test_attempts/attempt_1.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:greater.cc.test      FLAKY, failed in 1 out of 2 in 5.1s
  Stats over 2 runs: max = 5.1s, min = 4.9s, avg = 5.0s, dev = 0.1s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/greater.cc.test/test_attempts/attempt_1.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:max.cc.test          FLAKY, failed in 1 out of 2 in 8.1s
  Stats over 2 runs: max = 8.1s, min = 8.1s, avg = 8.1s, dev = 0.0s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/max.cc.test/test_attempts/attempt_1.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:min.cc.test          FLAKY, failed in 1 out of 2 in 7.7s
  Stats over 2 runs: max = 7.7s, min = 7.4s, avg = 7.6s, dev = 0.1s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/min.cc.test/test_attempts/attempt_1.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:softplus.cc.test     FLAKY, failed in 1 out of 2 in 5.5s
  Stats over 2 runs: max = 5.5s, min = 4.6s, avg = 5.1s, dev = 0.4s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softplus.cc.test/test_attempts/attempt_1.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:less.cc.test         FLAKY, failed in 2 out of 3 in 5.0s
  Stats over 3 runs: max = 5.0s, min = 5.0s, avg = 5.0s, dev = 0.0s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/less.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/less.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:sum.cc.test          FLAKY, failed in 2 out of 3 in 16.2s
  Stats over 3 runs: max = 16.2s, min = 14.1s, avg = 14.9s, dev = 0.9s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/sum.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/sum.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:maximum.cc.test     FAILED in 3 out of 3 in 5.4s
  Stats over 3 runs: max = 5.4s, min = 5.1s, avg = 5.2s, dev = 0.1s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/maximum.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/maximum.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/maximum.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:mean.cc.test        FAILED in 3 out of 3 in 8.7s
  Stats over 3 runs: max = 8.7s, min = 6.8s, avg = 7.7s, dev = 0.8s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/mean.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/mean.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/mean.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:not_equal.cc.test   FAILED in 3 out of 3 in 5.3s
  Stats over 3 runs: max = 5.3s, min = 4.6s, avg = 4.9s, dev = 0.3s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/not_equal.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/not_equal.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/not_equal.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:real_div.cc.test    FAILED in 3 out of 3 in 4.6s
  Stats over 3 runs: max = 4.6s, min = 3.9s, avg = 4.4s, dev = 0.3s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/real_div.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/real_div.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/real_div.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:softmax.cc.test     FAILED in 3 out of 3 in 4.8s
  Stats over 3 runs: max = 4.8s, min = 4.2s, avg = 4.5s, dev = 0.2s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:softmax_cross_entropy_with_logits.cc.test FAILED in 3 out of 3 in 8.0s
  Stats over 3 runs: max = 8.0s, min = 7.4s, avg = 7.7s, dev = 0.3s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax_cross_entropy_with_logits.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax_cross_entropy_with_logits.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/softmax_cross_entropy_with_logits.cc.test/test_attempts/attempt_2.log
//tensorflow/compiler/mlir/disc/tests/tensorflow_ops:sub.cc.test         FAILED in 3 out of 3 in 4.8s
  Stats over 3 runs: max = 4.8s, min = 4.3s, avg = 4.5s, dev = 0.2s
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/sub.cc.test/test.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/sub.cc.test/test_attempts/attempt_1.log
  /home/github/.cache/bazel/_bazel_github/ae143f62e65da17ca5d5f97d2dc3de17/execroot/org_tensorflow/bazel-out/k8-opt/testlogs/tensorflow/compiler/mlir/disc/tests/tensorflow_ops/sub.cc.test/test_attempts/attempt_2.log
```

--- a/.github/workflows/main.yml
+++ b/.github/workflows/main.yml
@@ -1,6 +1,6 @@
 # This is a basic workflow that is manually triggered
 
-name: DISC 
+name: DISC
 
 # Controls when the action will run. Workflow runs when manually triggered using the UI
 # or API.
@@ -16,7 +16,7 @@ jobs:
   CUDA10-TF115:
     # The type of runner that the job will run on
     runs-on: [self-hosted, gpu-t4]
-      
+
     # Steps represent a sequence of tasks that will be executed as part of the job
     steps:
     # Runs a single command using the runners shell
@@ -156,13 +156,12 @@ jobs:
         docker build -t disc-dev-cpu --build-arg BASEIMAGE=nvidia/cuda:10.0-cudnn7-devel-ubuntu18.04 -f docker/dev/Dockerfile .
     - name: Build And Test DISC
       run: |
-        # TODO(yancey.yx): enable CPU unit test
-        #set -e
-        #docker run --rm -t --user $(id -u) \
-        #  -v $HOME/.cache:$HOME/.cache \
-        #  -v /etc/passwd:/etc/passwd:ro \
-        #  -v /etc/group:/etc/group:ro \
-        #  -v $PWD:/disc \
-        #  -e GITHUB_WORKFLOW=$GITHUB_WORKFLOW \
-        #  -w /disc \
-        #  disc-dev-cpu bash ./scripts/ci/build_and_test.sh --cpu-only
+        set -e
+        docker run --rm -t --user $(id -u) \
+          -v $HOME/.cache:$HOME/.cache \
+          -v /etc/passwd:/etc/passwd:ro \
+          -v /etc/group:/etc/group:ro \
+          -v $PWD:/disc \
+          -e GITHUB_WORKFLOW=$GITHUB_WORKFLOW \
+          -w /disc \
+          disc-dev-cpu bash ./scripts/ci/build_and_test.sh --cpu-only

--- a/tao_compiler/mlir/disc/transforms/disc_outline_cpu_kernel.cc
+++ b/tao_compiler/mlir/disc/transforms/disc_outline_cpu_kernel.cc
@@ -188,7 +188,7 @@ FuncOp outlineKernelFunc(Operation* launchOp, StringRef kernelFnName,
   return funcOp;
 }
 
-StringRef getKernelName(int index, scf::ParallelOp op, StringRef funcName) {
+std::string getKernelName(int index, scf::ParallelOp op, StringRef funcName) {
   lmhlo::FusionOp fusion = op->getParentOfType<lmhlo::FusionOp>();
   auto strIndex =
       (index != 0 ? ("_" + llvm::Twine(index)) : llvm::Twine("")).str();
@@ -366,8 +366,7 @@ LogicalResult DiscOutlineCpuKernel::processFunction(SymbolTable& symbolTable,
     SmallVector<Value> operands;
     operands.push_back(func.getArgument(0));
 
-    auto kernelFnName =
-        getKernelName(en.index(), en.value(), func.getName()).str();
+    auto kernelFnName = getKernelName(en.index(), en.value(), func.getName());
 
     // Create launch setting operands (e.g. lower/upper/step) and rewrite
     // parallelOp accordingly. To wrap the targeting parallelOp, in order to

